cnn_model.py

```diff
@@ -0,0 +1,159 @@
+#%%
+from numpy.lib import index_tricks
+import torch, torchmetrics
+import time
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from typing import ClassVar
+from imgs_dataset import RHSImgDataset
+from torch.utils.data.sampler import SubsetRandomSampler
+from sklearn.metrics import confusion_matrix, ConfusionMatrixDisplay
+
+from torch.utils.tensorboard import SummaryWriter
+
+class RHSCNN(torch.nn.Module):
+    def __init__(self, n_classes, in_channels=3, negative_slope=0.01):
+        super().__init__()
+        self.conv_layers = torch.nn.Sequential(
```

Comment:
`self.conv_layers` = misnomer. This is not just the conv layers, it also includes the linear layers.